README.md

```
@@ -56,103 +36,38 @@ For a quicker dive into the idea, see the [**Website**](https://citadel-tech.git
 
-```console
-# Check binaries
-./docker-setup taker --help
-./docker-setup maker-cli --help
+Check the [demo doc](./docs/demo.md) for quick setup guides.
 
-# Test connection to market
-./docker-setup taker fetch-offers
-```
+> [!NOTE]
+> These apps should be considered as "examples", rather than products. We encourage all Bitcoin wallet developers to take a look at our [examples](https://github.com/citadel-tech/coinswap/tree/master/examples), [ffis](), and [apis](https://github.com/citadel-tech/coinswap/blob/master/src/taker/api.rs), to build their own apps, or integrate Coinswap within their existing apps. App ecosystem diversity is crucial for decentralization.
```

markdownlint-cli2 (0.22.1)

[warning] No empty links

(MD042, no-empty-links)